healthcare-management/src/app/app-config/app.config.ts

```diff
@@ -1,7 +1,7 @@
 
 import {AppConfig} from './app.config.interface';
 import {InjectionToken} from '@angular/core';
-import {environment} from '../../environments/environment';
+import {environment} from '../../environments/environment.development';
```

Comment:
Aici trebuie lasat environment simplu, deoarece in vercel se foloseste environment simplu